examples/geopmhash.c

```diff
@@ -3,19 +3,19 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
-#include <stdlib.h>
 #include <stdio.h>
+#include <stdlib.h>
 #include <string.h>
 #include "geopm_hash.h"
 
 int main(int argc, char **argv)
 {
-    const char *usage = "%s string\n\n"
-                        "    Returns the geopm_hash() of the input string.\n"
-                        "    this is useful for determining the region name\n"
-                        "    from the region_id printed in the trace.\n";
-    if (argc == 1 ||
-        strncmp("-h", argv[1], strlen("-h")) == 0 ||
+    const char *usage =
+        "%s string\n\n"
+        "    Returns the geopm_hash() of the input string.\n"
+        "    this is useful for determining the region name\n"
+        "    from the region_id printed in the trace.\n";
+    if (argc == 1 || strncmp("-h", argv[1], strlen("-h")) == 0 ||
```

Comment:
Not a fan of the unindent.

Reply:
Setting `AlwaysBreakBeforeMultilineStrings` to false may help achieve the desired behavior.

Reply:
I have updated the rule, in this case I think we are still bound by the 80 char limit, which I'm okay with.